tools/bpf/bpftool/Documentation/bpftool-feature.rst

```diff
@@ -19,19 +19,42 @@ SYNOPSIS
 FEATURE COMMANDS
 ================
 
-|	**bpftool** **feature probe** [*COMPONENT*] [**macros** [**prefix** *PREFIX*]]
+|	**bpftool** **feature probe** [*COMPONENT*] [**section** [*SECTION*]] [**filter_in** *PATTERN*] [**filter_out** *PATTERN*] [**macros** [**prefix** *PREFIX*]]
```

Comment:
If `section` is passed, then `SECTION` is not optional. We should have `[**section** *SECTION*]` instead of `[**section** [*SECTION*]]` I think? (Here and below)